README.md

```diff
@@ -81,6 +87,21 @@ loop {
 }
 ```
  
+ ### Tokio runtime
+
+Ideally, we should opt for making the SA implementation runtime-agnostic. However, at that stage the implementation is fully based on Tokio-rs/Runtime. That's said the recommended way of setting up the runtime is shown below.
```

Comment:
- Why should it be runtime-agnostic? If we can sum it up in few words, I'd mention it.
- At 'this' stage (or change it to 'currently')
- That's said --> That said,

Reply:
You're right. WRT `runtime-agnostic` idea, much better could be to file an issue about it instead of mentioning it in README.